<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AuthDialog</class>
 <widget class="QDialog" name="AuthDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>285</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Аутентификация</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>Войти</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="labelLogin">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Логин</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditLogin"/>
         </item>
         <item>
          <widget class="QLabel" name="labelPassword">
           <property name="font">
            <font>
             <pointsize>12</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>Пароль</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEditPassword"/>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxShowPassword">
           <property name="text">
            <string>Показать пароль</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxSavePassword">
           <property name="text">
            <string>Сохранить пароль</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonLogin">
           <property name="text">
            <string>Войти</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Зарегистрироваться</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <item>
          <layout class="QGridLayout" name="gridLayout">
           <item row="2" column="0">
            <widget class="QLabel" name="labelPassword_3">
             <property name="text">
              <string>Повторите пароль</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="lineEditLoginSignup"/>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="labelLogin_2">
             <property name="text">
              <string>Логин</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="labelPassword_2">
             <property name="text">
              <string>Пароль</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="labelEmail">
             <property name="text">
              <string>Email</string>
             </property>
            </widget>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="labelPhone">
             <property name="text">
              <string>Телефон</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="lineEditPasswordSignup">
             <property name="clearButtonEnabled" stdset="0">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="lineEditPasswordSignupRepeat"/>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="lineEditEmail"/>
           </item>
           <item row="4" column="1">
            <widget class="QLineEdit" name="lineEditPhone"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QCheckBox" name="checkBoxShowPassworsSignup">
           <property name="text">
            <string>Показать пароль</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pushButtonSignup">
           <property name="text">
            <string>Зарегистрироваться</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
